### PR TITLE
Fix Storage test-resources config - add one extra year

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

**Review by RickWinter — APPROVED** [review]

File: sdk/storage/test-resources.json [hunk 1]
```diff
@@ -337,4 +353,4 @@
             "value": "[concat('rg-storage-', variables('accountName'))]"
         }
     }
-}
+}
```
Comment (Jinming-Hu) [inline sdk/storage/test-resources.json:356]: new line

**Review by Jinming-Hu — APPROVED** [review]

File: sdk/storage/test-resources.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
             "signedPermission": "rwdlacup",
             "signedResourceTypes": "sco",
             "keyToSign": "key2",
-            "signedExpiry": "2022-01-01T23:59:00Z"
+            "signedExpiry": "2023-01-01T23:59:00Z"
```
Comment (Jinming-Hu) [inline sdk/storage/test-resources.json:61]: How about just 2099? so that we don't have to fix it every year

Comment (vhvb1989) [reply sdk/storage/test-resources.json:61]: not sure if that would be valid.. Usually the Azure Portal lets to set 1 year the most..  but let's try it

Comment (vhvb1989) [reply sdk/storage/test-resources.json:61]: let's use [this PR](https://github.com/Azure/azure-sdk-for-cpp/pull/3210) in case 2099 works

Comment (Jinming-Hu) [reply sdk/storage/test-resources.json:61]: > not sure if that would be valid.. Usually the Azure Portal lets to set 1 year the most.. but let's try it

I tried on Azure Portal. I can generate a SAS token expires in 2099.

Comment (vhvb1989) [reply sdk/storage/test-resources.json]: beautiful!  I have updated to be 99 . Thank you

**Pull request merged**